compute/instances/reference-content/add-instance-specific-ssh-keys-using-tags.mdx

```diff
@@ -0,0 +1,106 @@
+---
+meta:
+  title: Using tags to add Instance-specific SSH keys
+  description: Find out how to use tags to add Instance-specific SSH keys on Scaleway.
+content:
+  h1: Using tags to add Instance-specific SSH keys
+  paragraph: Find out how to use tags to add Instance-specific SSH keys on Scaleway.
+categories:
+  - compute
+dates:
+  validation: 2024-10-08
+tags: Instance ssh-key ssh tag
+---
+
+In cloud environments, managing SSH keys across multiple Instances is key to keeping your infrastructure secure and easy to access.
+Sometimes, you need to ensure that a specific SSH key grants access to only one Instance and does not automatically apply to others.
+This can simplify key management and improve security, especially when you are running many Instances
+
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
+- An [Instance](/compute/Instances/how-to/create-an-Instance/)
+
+## The authorized_keys file
+
+Your authorized SSH keys are stored in the `/root/.ssh/authorized_keys` file on your Instance.
+If you inspect this file, you might notice a small comment mentioning the use of tags to add SSH keys directly to Instances.
+
+```bash
+cat /root/.ssh/authorized_keys
+#
+# WARNING: Automatically generated file
+# This file will be erased at every boot
+# This file was generated with '/usr/sbin/scw-fetch-ssh-keys'
+#
+# To ad
```

Comment:
Probably not up to us, but it would be nice to rewrite the template a bit as it has a typo and the procedure is not very clear

Reply:
I agree that this template uses an outdated URL, but it has never been updated. 
I will ask them where I can find it